### PR TITLE
added optaweb-vehicle-routing

### PR DESCRIPTION
**Pull request by mbiarnes**
added optaweb-vehicle-routing to pr jobs and deployed jobs

### CODE REVIEW

File: job-dsls/jobs/deploy_jobs.groovy [hunk 1]
```diff
@@ -158,12 +158,17 @@ def final REPO_CONFIGS = [
...
         "optaweb-employee-rostering" : [
                 ircNotificationChannels: ["#optaplanner-dev"],
                 artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
                         "**/target/configurations/cargo-profile/profile-log.txt"
                 ]
+        ],
+        "optaweb-vehicle-routing" : [
+                ircNotificationChannels: ["#optaplanner-dev"],
+                artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
+                        "**/target/configurations/cargo-profile/profile-log.txt"
```
Comment (yurloc) [inline job-dsls/jobs/deploy_jobs.groovy]: Nope. Cargo is not used in this repository. Let's keep just defaults.

File: job-dsls/jobs/pr_jobs.groovy [hunk 2]
```diff
@@ -63,6 +63,11 @@ def final REPO_CONFIGS = [
                         "**/target/configurations/cargo-profile/profile-log.txt"
                 ]
         ],
+        "optaweb-vehicle-routing" : [
+                artifactsToArchive     : DEFAULTS["artifactsToArchive"] + [
+                        "**/target/configurations/cargo-profile/profile-log.txt"
```
Comment (yurloc) [inline job-dsls/jobs/pr_jobs.groovy]: Here too.

Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy]: removed

File: job-dsls/jobs/deploy_jobs.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/deploy_jobs.groovy]: removed

**Pull request merged**